[2024-04-22T18:16:53.467+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-22T18:16:53.508+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-22T18:16:53.522+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-22T18:16:53.523+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-22T18:16:53.534+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): task1> on 2024-04-15 00:00:00+00:00
[2024-04-22T18:16:53.544+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1080) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-22T18:16:53.543+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_backfill_v02', 'task1', 'backfill__2024-04-15T00:00:00+00:00', '--job-id', '395', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpd4v0o9r8']
[2024-04-22T18:16:53.546+0000] {standard_task_runner.py:63} INFO - Started process 1082 to run task
[2024-04-22T18:16:53.547+0000] {standard_task_runner.py:91} INFO - Job 395: Subtask task1
[2024-04-22T18:16:53.635+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-04-15T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-22T18:16:53.741+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='dag_with_catchup_backfill_v02' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-04-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-04-15T00:00:00+00:00'
[2024-04-22T18:16:53.748+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-22T18:16:53.761+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-22T18:16:53.762+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-04-22T18:16:53.772+0000] {subprocess.py:86} INFO - Output:
[2024-04-22T18:16:53.773+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-04-22T18:16:53.774+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-22T18:16:53.777+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-22T18:16:53.796+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v02, task_id=task1, execution_date=20240415T000000, start_date=20240422T181653, end_date=20240422T181653
[2024-04-22T18:16:53.820+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-22T18:16:53.836+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-22T18:16:53.840+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
